rpc/src/history.rs

```diff
@@ -190,7 +190,10 @@ impl HistoryNetworkApiServer for HistoryNetworkApi {
     async fn local_content(&self, content_key: HistoryContentKey) -> RpcResult<HistoryContentItem> {
         let endpoint = HistoryEndpoint::LocalContent(content_key);
         let result = self.proxy_query_to_history_subnet(endpoint).await?;
-        let result: HistoryContentItem = from_value(result)?;
+        let result: HistoryContentItem = match result {
+            Value::Null => HistoryContentItem::Unknown(String::from("")),
```

Comment:
@ogenev I made [changes](https://github.com/jacobkaufmann/trin/blob/57b8afbbc8c9fc80b76faea5aaae0d2adca12103/ethportal-api/src/types/content_item.rs#L267-L285) to the `serde` traits so that `0x0` would be serialized for an empty `Unknown` value. but like I said in [the issue on `Unknown`](https://github.com/ethereum/trin/issues/538#issuecomment-1437400673), `Unknown` is not a good representation for a null value because it complicates handling these cases.

Reply:
I see, I'm ok leaving it as it is and eventually tackling #538 in a future PR.